code/modules/medical/genetics/bioEffects/beneficial.dm

```diff
@@ -310,6 +310,29 @@
 	icon_state  = "dead"
 	isBad = 1
 
+/datum/bioEffect/noir
+	name = "Noir"
+	desc = "The subject generates a light-defying aura, equalizing photons in such a way that make them look completely grayscale."
+	id = "noir"
+	effectType = EFFECT_TYPE_POWER
+	probability = 99
+	stability_loss = 5
+	icon_state  = "noir"
+	msgGain = "You feel chromatic pain."
+	msgLose = "Colors around you begin returning to normal."
+
+	OnAdd()
+		..()
+		if (ishuman(owner))
+			var/mob/living/carbon/human/H = owner
+			animate_fade_grayscale(H, 5)
```

Comment:
As it is now I think this will get removed if the player reconnects to the game.

Reply:
Can't test it in VSC, it closes the whole host 'cause the main window closes. I'd say roll with it, idk.

Reply:
I looked at the current noir effect code and it seems like this is already taken care of.